sycl/test-e2e/InvokeSimd/Spec/simd_size/simd32.cpp

```diff
@@ -1,12 +1,9 @@
-// Test intended to run on PVC only
-// REQUIRES: gpu-intel-pvc
-//
 // Check that full compilation works:
 // RUN: %{build} -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr -o %t.out
-// RUN: env IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %{run} %t.out
+// RUN: env IGC_ForceOCLSIMDWidth=32 IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %{run} %t.out
```

Comment:
im not sure i understand why we need to pass this, shouldnt the subgroup size be determined to be 32 automatically, and the subgroup size has to equal the simd size?